### DISCUSSION TITLE
Generating SVG Legend?

### DISCUSSION THREAD

**Discussion by rpivovar-godaddy** [observablehq/plot#869] [Q&A] [answered]
If we want to generate a legend that is also a part of the SVG (as opposed to HTML wrapping the SVG), would we have to manually do this, like with several Plot.bars and Plot.texts, for instance?

**Comment by Fil** [comment, marked as answer]
There are many ways to do this, see https://observablehq.com/@fil/plot-a-legend-embedded-in-a-chart for a (not so good) example. Doing generic page layout in svg is a bit of a challenge.
